torchao/quantization/prototype/mixed_precision/scripts/naive_intNwo.py

```diff
@@ -0,0 +1,40 @@
+import torch
+
+from torchao.quantization.quant_primitives import (
+    MappingType,
+    ZeroPointDomain,
+)
+
+def intN_weight_only_asym(group_size=32, n=8):
```

Comment:
Can you add a short docstring to describe what this is doing? Maybe add an example to use this with the `quantize_` API? (same for `intN_weight_only_sym`)

Reply:
also, should we limit this to n = [2, 3, 4, 5, 6, 8] for now? (throw error otherwise)

Reply:
Added the doctoring and assertion to limit [2,3,4,5,6,8] only